### PR TITLE
Describe new miqldap-to-sssd arguments.

### PR DESCRIPTION
**Pull request by jvlcek**
Arguments bind-dn and bind-pwd were added to miqldap_to_sssd
in PR: https://github.com/ManageIQ/manageiq/pull/16979

This PR updates the miqldap-to-sssd blog post to include
descriptions of these two new arguments.

### CODE REVIEW

**Comment by jvlcek** [comment]
@miq-bot add_label blog post

**Comment by jvlcek** [comment]
@miq-bot add_label "blog post"

**Comment by jvlcek** [comment]
I added the two new arguments bind-dn and bind-pwd and corrected some spelling errors.

@cybette and @djberg96  Please review

**Comment by jvlcek** [comment]
@miq-bot assign @abellotti

**Comment by djberg96** [comment]
LGTM 👍 

Looks like you took this opportunity to fix some other spelling mistakes as well. :)

File: site/_posts/2017-09-29-miqldap-to-sssd.md [hunk 1]
```diff
@@ -164,28 +165,32 @@ Screenshot showing the appliance configuration for Mode: LDAP:
...
-*miqldap_to_sssd* supports 5 arguments. This example uses the **--basedn-domain** argument.
+*miqldap_to_sssd* supports 7 arguments. This example uses the **basedn-domain**, **bind-dn** and **bind-pwd** arguments.
 <pre><code style="white-space: pre">
 miqldap_to_sssd --help
 Usage: ruby tools/miqldap_to_sssd.rb [options]
-  -d, --basedn-domain=                       The Base DN domain name, e.g. example.com
-  -c, --tls-cacert=                          Path to certificate file
+  -d, --basedn-domain                        The Base DN domain name, e.g. example.com
+  -b, --bind-dn                              The Bind DN, credential to use to authenticate against LDAP e.g. cn=Manager,dc=example,dc=com
+  -p, --bind-pwd                             The Base DN domain name, e.g. example.com
```
Comment (cybette) [inline site/_posts/2017-09-29-miqldap-to-sssd.md]: The description doesn't seem to match the argument.

File: site/_posts/2017-09-29-miqldap-to-sssd.md [hunk 2]
```diff
@@ -227,26 +232,29 @@ Screenshot showing the appliance configuration for Mode: LDAP:
...
 preserves the certificate security.
 
 <pre><code style="white-space: pre">
 miqldap_to_sssd --help
 Usage: ruby tools/miqldap_to_sssd.rb [options]
-  -d, --basedn-domain=                       The Base DN domain name, e.g. example.com
-  -c, --tls-cacert=                          Path to certificate file
+  -d, --basedn-domain                        The Base DN domain name, e.g. example.com
+  -b, --bind-dn                              The Bind DN, credential to use to authenticate against LDAP e.g. cn=Manager,dc=example,dc=com
+  -p, --bind-pwd                             The Base DN domain name, e.g. example.com
```
Comment (cybette) [inline site/_posts/2017-09-29-miqldap-to-sssd.md]: as above

File: site/_posts/2017-09-29-miqldap-to-sssd.md (hunk 1 above)
Comment (jvlcek) [reply site/_posts/2017-09-29-miqldap-to-sssd.md]: @cybette Thank you for the reply.  I don't quite understand you comment
tls-cacert is the argument where the certificate file path can be specified.

Do you want me to change the name of the flag to include `file`?

e.g.: --tls-cacert-file

Comment (cybette) [reply site/_posts/2017-09-29-miqldap-to-sssd.md]: @jvlcek I'm referring to `bind-pwd` with the description "The Base DN domain name, e.g. example.com"

Comment (jvlcek) [reply site/_posts/2017-09-29-miqldap-to-sssd.md]: @cybette AH! Yes. I see it now! Thank you. I will fix it.

**Comment by abellotti** [comment]
Hi @jvlcek, I'm good in general with the doc.  Other than the bind-pwd description of the Bind Password parameter that @cybette noted above, I'm not crazy about the basedn-domain option name. While I see the internal logic generating the domain based on a basedn if not specified, the parameter itself is just the domain. We're not expecting a basedn "<rdn>=....,<rdn>=..." input from the user, just the FQDN.  So maybe the option name should just be that --domain. This would require code change though so maybe as separate PRs. Thanks.

**Comment by cybette** [comment]
@jvlcek LGTM. Will there be code change corresponding to the option name change as @abellotti mentioned? If so maybe you can link to that PR and I can merge this to sync with the code change.

**Comment by jvlcek** [comment]
Thank you @abellotti and @cybette!

I have made the corrections you have suggested.
Please do not merge this PR until [PR 17009](https://github.com/ManageIQ/manageiq/pull/17009), for code changes needed to address your suggestions, has been merged.

**Comment by jvlcek** [comment]
@cybette Yes there are code changes. You are just too good and fast and did your review while I was adding the comment about them! :) Thank you!

**Comment by jvlcek** [comment]
@cybette This should be good to merge now that PR [17009](https://github.com/ManageIQ/manageiq/pull/17009)  has been merged.

@djberg96 @abellotti @cybette Thank you for the 👀  !

**Pull request merged**